packages/@ama-sdk/create/src/index.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+import { spawnSync } from 'node:child_process';
+import { resolve } from 'node:path';
+import * as minimist from 'minimist';
+
+const defaultScope = 'sdk';
+const binPath = resolve(require.resolve('@angular-devkit/schematics-cli/package.json'), '../bin/schematics.js');
+const args = process.argv.slice(2);
+const argv = minimist(args);
+
+if (argv._.length === 0) {
+  // eslint-disable-next-line no-console
+  console.error('The project name is mandatory');
+  process.exit(-1);
+}
+
+let [name, pck] = argv._[0].replace(/^@/, '').split('/');
```

Comment:
shouldn't this be `[pck, name]` ? or am I reading it wrong ?

Reply:
nvm, `name` seems to be the npm package scope so it's fine, but it reads weird

Reply:
I kept the naming of the generator options, I agree, it is weird :s